### ISSUE TITLE
Pasting with multiple selections pastes too many instances of clipboard contents

### ISSUE THREAD

**Issue by mdonatas** [dbeaver/dbeaver#21230]
### Description

When pasting with multiple selections active, each position with a cursor gets as many copies of clipboard content as there are cursors.
So 2 cursors would get 2 * 2 = 4 copies of clipboard pasted
And 5 would get 5 * 5 = 25

### DBeaver Version

Community Edition 23.2.0

### Operating System

Windows 11 Pro

### Steps to reproduce

- Select multiple occurrences of something
- Paste

Here is a demo where instead of pasting the contents 3 times (as one would expect) it gets pasted 9 times
![multi-select-paste](https://github.com/dbeaver/dbeaver/assets/483659/8069ca10-5d1f-49b1-a54f-520a176842a6)

**Comment by E1izabeth** [comment]
I reproduced.

**Comment by E1izabeth** [comment]
@mdonatas thank you for bug report.

**Comment by MashaKorax** [comment]
Did you use some external plugins?

**Comment by mdonatas** [comment]
> Did you use some external plugins?

No, default installation. Until this question I didn't even know dbeaver had them 😅 I've used this feature https://stackoverflow.com/a/75113497/79981

**Comment by DavidHJong** [comment]
problem still exists for Community Ediction 25.0.1. Any idea for debugging?

**Comment by sonneander** [comment]
This would solve my hopping back and fourth between VS Code and DBeaver. High DX improvement for sure.
